client/pages/Login/index.jsx

```diff
@@ -1,14 +1,30 @@
 // React lib import
 import React from 'react';
 
+// third-party imports
+import { Redirect } from 'react-router-dom';
+import dotenv from 'dotenv';
+
+// utils
+import isLoggedIn from '../../utils/isLoggedIn';
+
+
+dotenv.config();
 
 /**
  * Renders the Login component
  *
  * @returns {JSX} JSX
  * @memberof Login
  */
-const Login = () => (
+const Login = (props) => {
+  const redirectUrl = `${process.env.ANDELA_API_BASE_URL}/login?redirect_url=${process.env.BASE_URL}`;
```

Comment:
I think it's high time we have a global config object where we can all the variables we need. Similar to how it was on learning-map.